src/main/java/uk/gov/hmcts/darts/arm/service/impl/AbstractArmBatchProcessResponseFiles.java

```diff
@@ -317,6 +325,73 @@ && nonNull(armResponseBatchData.getUploadFileFilenameProcessor())) {
+                        String errorCode = getErrorCode(invalidLineRecord1, invalidLineRecord2);
+                        externalObjectDirectory.setErrorCode(errorCode);
+                        updateExternalObjectDirectoryStatus(externalObjectDirectory, EodHelper.armResponseManifestFailedStatus());
+
+                    } else {
+                        log.warn("Unable to read invalid line files {} - {}", invalidLineFileFilenameProcessor1.getInvalidLineFileFilenameAndPath(),
+                                 invalidLineFileFilenameProcessor2.getInvalidLineFileFilenameAndPath());
+                        updateExternalObjectDirectoryStatus(externalObjectDirectory, EodHelper.armResponseProcessingFailedStatus());
+                    }
+                } else {
+                    log.warn("Invalid line file count is greater than 2 for external object directory ID {}", externalObjectDirectory.getId());
+                    updateExternalObjectDirectoryStatus(externalObjectDirectory, EodHelper.armResponseProcessingFailedStatus());
+                }
+            } else {
+
+                log.warn("Unable to find external object directory with ID {} for ARM batch responses with first IL file {}, second IL file {}",
+                         armResponseBatchData.getExternalObjectDirectoryId(),
+                         invalidLineFileFilenameProcessor1.getInvalidLineFileFilenameAndPath(),
+                         invalidLineFileFilenameProcessor2.getInvalidLineFileFilenameAndPath());
+                List<String> invalidResponseFiles = new ArrayList<>();
+                invalidResponseFiles.add(invalidLineFileFilenameProcessor1.getInvalidLineFileFilenameAndPath());
+                invalidResponseFiles.add(invalidLineFileFilenameProcessor2.getInvalidLineFileFilenameAndPath());
+                deleteResponseBlobs(invalidResponseFiles);
+            }
+        } catch (Exception e) {
+            log.error("Unable to update invalid line responses", e);
+        }
+
+    }
+
+    private String getErrorCode(ArmResponseInvalidLineRecord invalidLineRecord1, ArmResponseInvalidLineRecord invalidLineRecord2) {
+        UploadNewFileRecord uploadNewFileRecord1 = readInputJson(invalidLineRecord1.getInput());
+        UploadNewFileRecord uploadNewFileRecord2 = readInputJson(invalidLineRecord2.getInput());
+
+        String errorCode;
+        if (CREATE_RECORD.equalsIgnoreCase(uploadNewFileRecord1.getOperation())) {
+            errorCode = invalidLineRecord1.getExceptionDescription() + " " + invalidLineRecord2.getExceptionDescription();
+        } else if (CREATE_RECORD.equalsIgnoreCase(uploadNewFileRecord2.getOperation())) {
+            errorCode = invalidLineRecord2.getExceptionDescription() + " " + invalidLineRecord1.getExceptionDescription();
+        } else {
+            errorCode = invalidLineRecord1.getExceptionDescription() + " " + invalidLineRecord2.getExceptionDescription();
+        }
+        return errorCode;
```

Comment:
Can be condensed
`if (CREATE_RECORD.equalsIgnoreCase(uploadNewFileRecord2.getOperation())) {
            return invalidLineRecord2.getExceptionDescription() + " " + invalidLineRecord1.getExceptionDescription();`
            }
            return errorCode = invalidLineRecord1.getExceptionDescription() + " " + invalidLineRecord2.getExceptionDescription();`

Reply:
Sorry that was a type in the else if statement.